neptune/exceptions.py

```diff
@@ -13,18 +13,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import warnings
+
 from neptune import envs
 
+STYLES = {'h1': '\033[95m',
+          'h2': '\033[94m',
+          'python': '\033[96m',
+          'bash': '\033[95m',
+          'warning': '\033[93m',
+          'correct': '\033[92m',
+          'fail': '\033[91m',
+          'bold': '\033[1m',
+          'underline': '\033[4m',
+          'end': '\033[0m'}
+
 
 class NeptuneException(Exception):
     pass
 
 
-class Uninitialized(NeptuneException):
+class NeptuneUninitializedException(NeptuneException):
     def __init__(self):
-        super(Uninitialized, self).__init__(
-            "You must initialize neptune-client first. "
-            "For more information, please visit: https://github.com/neptune-ai/neptune-client#initialize-neptune")
+        message = """
+{h1}     
+----NeptuneUninitializedException---------------------------------------------------------------------------------------
+{end}
+You must initialize neptune-client before you create an experiment.
+
+Looks like you forgot to add:
+    {python}neptune.init(api_token='YOUR_LONG_TOKEN', project_qualified_name='WORKSPACE_NAME/PROJECT_NAME'){end}
+    
+before you ran:
+    {python}neptune.create_experiment(){end}
+    
+You may also want to check the following docs pages:
+    - https://docs.neptune.ai/getting-started/quick-starts/log_first_experiment.html
+    
+{correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help.html
+""".format(**STYLES)
+        super(NeptuneUninitializedException, self).__init__(message)
+
+
+def Uninitialized():
```

Comment:
This won't work, since you can't do this:
```py3
try:                                                                                                                                                                                                               
    raise exceptions.NeptuneUninitializedException()                                                                                                                                                               
except exceptions.Uninitialized as e:                                                                                                                                                                              
    print(e)                                                                                                                                                                                                       
```
because of
```
TypeError: catching classes that do not inherit from BaseException is not allowed
```

Also, we're using `Uninitialized` in our code, so every time we raise it, user gets a warning they can do nothing about.

If you feel we need the rename, please change the usages in our code as well, but still, `Uninitialized` cannot be left as a function.

I think the most we can do here is something like `Uninitialized = NeptuneUninitializedException`, but I'm not sure how to make our users aware of the deprecation. Maybe it's not required? How many people are actually catching this particular exception?

Basically, those exceptions are thrown only by our lib. The only way our users should use them is to catch them. And I'm not sure anyone would bother to catch exceptions that you're changing here.

Reply:
As I am using this name in the error message I'd rather have it mention Neptune.
That is why I wanted to rename.

I went with the rename in code also option.
Is this ok?